content/docs/tina-cloud/beta-migration.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Upgrading from Alpha to Beta
+---
+
+## Moving Tina Cloud to Beta
+
+Thanks for participating in the Tina Cloud Alpha! Because of you we were able to gather a ton of useful feedback, identify pain points, 
+and get a good idea about how real people will use Tina Cloud in the future.
+
+We've made a few changes and are ready to make a Tina Cloud beta publicly available. 
+
+## Beta Changes
+
+Aside from some UI changes that you've might have noticed us developing in the Tina Cloud Dashboard, we've also been making a bunch of changes 
+under the hood. One of the biggest of these changes is how we handle user accounts. 
+
+Previously when you created a Tina Cloud account you were creating an **organization** that you accessed at `https://OrgName.tina.io`. We thought that giving each organization it's own unique subdomain and pool of users would be neat, but after spending some time in that paradigm it turned out to be a lot of added complexity that didn't add much value. 
+
+With the release of the beta we're moving user accounts to the forefront. Whenever you create a Tina Cloud account you'll always access it at https://app.tina.io. This means that even
+if you are part of multiple organizations, you'll only have a single Tina account.
+
+> Tools for creating/managing organizations within Tina Cloud accounts are coming. For now we create a default organization, which you'll still be able to invite users to.
+
+
+## Migration Path
+
+Since we're changing how user accounts work within Tina Cloud, alpha users will need to create new accounts for the beta. The process is pretty simple, you'll just need to **recreate your Tina Cloud account** an **update your Tina Cloud Site**. The good news is that since we don't control your content, there really isn't too much to do. 
```

Comment:
```suggestion
Since we're changing how user accounts work within Tina Cloud, existing alpha users must **recreate their Tina Cloud account** and **update their Tina Cloud Site**. The good news is that since we don't control your content, there really isn't anything else to do. 
```